test/ConfigAggregatorTest.php

```diff
@@ -92,10 +92,61 @@ function () {
         ], $this->cacheFile);
         $this->assertTrue(file_exists($this->cacheFile));
         $cachedConfig = include $this->cacheFile;
+
         $this->assertInternalType('array', $cachedConfig);
         $this->assertEquals(['foo' => 'bar', ConfigAggregator::ENABLE_CACHE => true], $cachedConfig);
     }
 
+    public function testConfigAggregatorCanCacheConfigWithClosures()
```

Comment:
Assuming we add the `CLOSURE_SNAPSHOT_USES` option, we should have another test to validate that a closure that uses binding is serialized correctly.

Reply:
Added testcase.